docs/en/observability/profiling-self-managed-ops.asciidoc

```diff
@@ -262,3 +263,85 @@ Download the corresponding binary version and replace the existing one, using th
 Replace the old binary and restart the services.
 
 You will find the links to the new binaries in the "Add Data" page, under the "Binary" tab.
+
+[discrete]
+[[profiling-self-managed-kubernetes-tips]]
+== Kubernetes tips
+
+When deploying the Universal Profiling backend on Kubernetes, there are some best practices to follow.
+
+[discrete]
+=== Ingress configuration
+
+If you are using an ingress controller, the connection routing to the collector Service should be configured to use the gRPC protocol.
+
+We provide an `Ingress` resource as part of the Helm chart, and because the ingress can be any implementation,
+you should provide to the chart values the configuration of the ingress, adding the class name and the
```

Comment:
> you should provide to the chart values the configuration of the ingress

I'm not sure what you're trying to say here.

Reply:
The Ingress is managed by the chart, but its configurations need to be passed through from the users, as they are implementation-specific.
This practically means that all the times an ingress is enabled, it must be configured with some values passed in Helm by users.

Reply:
Thanks, that's very helpful. I made a small recommendation based off of this comment: https://github.com/elastic/observability-docs/pull/3932/files#r1621192289